src/controlnet_aux/mlsd/__init__.py

```diff
@@ -38,7 +38,11 @@ def to(self, device):
         self.model.to(device)
         return self
     
-    def __call__(self, input_image, thr_v=0.1, thr_d=0.1, detect_resolution=512, image_resolution=512, return_pil=True):
+    def __call__(self, input_image, thr_v=0.1, thr_d=0.1, detect_resolution=512, image_resolution=512, return_pil=None, output_type="pil"):
```

Comment:
Would do the same changes as in https://github.com/patrickvonplaten/controlnet_aux/pull/49/files#r1214565652

To deprecate a function argument we must remove it from the signature (can't have both `return_pil` and `output_type=="pil"` in the signature at once - too confusin gto read